### ISSUE TITLE
Add option to the windows uninstaller for deleting the ~/pairinteraction folder

### ISSUE THREAD

**Issue by seweber** [pairinteraction/pairinteraction#61]
The user should be able to specify whether he likes to delete the ~/pairinteraction folder as well.

**Comment by hmenke** [comment]
This is tricky because the user uninstalling pairinteraction is not necessarily the same as the one who was using it.  The uninstaller would have to scan all user folders and remove the `~/pairinteration` folder.  We should document in the uninstaller that the folder has to be removed rather than trying to remove it ourselves.  Also I it is very unsafe to delete folders we did not create on installation!

**Comment by seweber** [comment]
Yes, it is a bit tricky. However, I think it could be done safely.

For obtaining the user folders, we could use [EnumUsersReg](http://nsis.sourceforge.net/EnumUsersReg) as described in http://forums.winamp.com/showthread.php?t=238043 or https://nsis-dev.github.io/NSIS-Forums/html/t-290045.html. After the uninstaller has found all the ~/pairinteration folders, the uninstaller should ask the user whether he allows to delete them.

By letting the pairinteraction program place some characteristic file inside the ~/pairinteraction folders or choosing a characteristic folder name, we could ensure that all the detected ~/pairinteraction folders were created by the program. We could also place the settings folder in the appdata directory.
